.gitignore

```
@@ -5,7 +5,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
-
+/**/**/node_modules
+/**/**/.cache
+/**/**/dist
```

```suggestion
node_modules
.cache
dist
```
이렇게 폴더 이름만 명시해도 된답니다 ㅎㅎ